.github/workflows/test.yml

```diff
@@ -72,16 +72,6 @@ jobs:
         run: |
           gem install bundler -v '~> 1.17.3'
           bundle update
```

Comment:
We run `bundler-cache`, so we can drop this bit I think. And also upgrade this test to Ruby 3.2